src/main/webapp/js/instructorFeedbackResults.es6

```diff
@@ -140,11 +140,6 @@ function filterResults(rawSearchText) {
     }
 }
 
-function updateResultsFilter() {
-    $('input[id=filterTextForDownload]').val($('#results-search-box').val());
-    filterResults($('#results-search-box').val());
```

Comment:
Already removed as part of that commit: [https://github.com/TEAMMATES/teammates/pull/7313/commits/7432c87770038f91605e35489b04edb001680d23#diff-8582992252efae5107e2b40ec07b54a6](https://github.com/TEAMMATES/teammates/pull/7313/commits/7432c87770038f91605e35489b04edb001680d23#diff-8582992252efae5107e2b40ec07b54a6l)